sentry-core/build.gradle.kts

```diff
@@ -65,6 +66,18 @@ tasks {
     }
 }
 
+buildConfig {
+    useJavaOutput()
+    packageName("io.sentry.core")
```

Comment:
It's package as in package-manager:

```suggestion
    packageName("maven:io.sentry.sentry-core")
```

Reply:
Would change when we rename to `io.sentry`

Reply:
This is a package where the plugin generates a class.